prime-router/src/main/kotlin/cli/tests/BasicTests.kt

```diff
@@ -175,6 +175,11 @@ class End2EndUniversalPipeline : CoolTest() {
             }
             translateReportIds.forEach { translateReportId ->
                 val batchResults = pollForStepResult(translateReportId, TaskAction.batch)
+                if (batchResults.isEmpty()) {
```

Comment:
Since the size of `batchResults` is never checked, the for loop on 184L, will happily pass even if there are no results which should be an error.